stories/notification/notification.stories.js

```diff
@@ -5,512 +5,295 @@ import '../../dist/notification.css';
+            <h2>Your leave request has been rejected</h2>
             </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            <p>SAP Analytics Cloud &middot; 7 minutes ago</p>
         </div>
-    </div>
-    <div class="fd-notification fd-notification--warning fd-notification--m">
-        <div class="fd-notification__header">
-            <div class="fd-notification__indicator--warning"></div>
-            <div class="fd-notification__title">حقيقة وأساس تلك السعادة البشرية، فلا أحد يرفض أو يكره أو يتجنب الشعور بالسعادة، ولكن بفضل هؤلاء الأشخاص الذين لا يدركون بأن السعادة لا بد أن نستشعرها بصورة أكثر عقلانيأسي.</div>
-            <button aria-label="decline-button" role="button" class="fd-button fd-button--transparent fd-notification__close">
+        <div class="fd-notification__actions">
+            <button class="fd-button fd-button--compact">Open</button>
+            <button class="fd-button fd-button--transparent fd-button--compact" aria-label="Close">
                 <i class="sap-icon--decline"></i>
             </button>
         </div>
-        <div class="fd-notification__body">
-            <div class="fd-notification__content">
-                <div class="fd-notification__avatar">
-                    <span class="fd-avatar fd-avatar--xs fd-avatar--circle" aria-label="Wendy Wallace">WW</span>
-                </div>
-                <div class="fd-notification__text">
-                    <div class="fd-notification__description">
-                       حقيقة وأساس تلك السعادة البشرية، فلا أحد يرفض أو يكره أو يتجنب الشعور بالسعادة، ولكن بفضل هؤلاء الأشخاص الذين لا يدركون بأن السعادة لا بد أن نستشعرها بصورة أكثر عقلانيأسي.
-                    </div>
-                    <div class="fd-notification__metadata">
-                       حقيقة وأساس تلك السعادة البشرية، فلا أحد يرفض أو يكره أو يتجنب الشعور بالسعادة، ولكن بفضل هؤلاء الأشخاص الذين لا يدركون بأن السعادة لا بد أن نستشعرها بصورة أكثر عقلانيأسي.
-                    </div>
-                </div>
-            </div>
-            <div class="fd-notification__footer">
-                <button role="button" class="fd-button fd-button--transparent">المزيد</button>
-                <div class="fd-notification__actions">
-                    <button role="button" class="fd-button fd-button--positive">موافق</button>
-                    <button role="button" class="fd-button fd-button--negative">إلغاء</button>
-                </div>
-            </div>
-        </div>
+    </div>
+    <div class="fd-notification__limit">
+        <h1>There are 30 more notifications</h1>
+        <p class="fd-notification__limit--description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
     </div>
 </div>
 `;
-RTL.parameters = {
+notificationGroup.parameters = {
```

Comment:
**DESIGN**
I'm noticing an ever-so-slight space missing between the Accept All button and the line separating the group header from the list items:

![Screen Shot 2020-09-03 at 10 28 39 AM](https://user-images.githubusercontent.com/57453945/92128317-4718a780-edd0-11ea-9d21-3fc49024f681.png)

**These are screenshots taken from the Visual Core specs:**

![Screen Shot 2020-09-03 at 10 02 55 AM](https://user-images.githubusercontent.com/57453945/92126976-d4f39300-edce-11ea-8007-0cfe94e0a792.png)

You can see the spacing more clearly in the HCB view:
![Screen Shot 2020-09-03 at 10 13 36 AM](https://user-images.githubusercontent.com/57453945/92127071-f05e9e00-edce-11ea-9e08-ab402342208e.png)

But from what I see in the specs, the spacing is 0rem... @droshev what do you think?

Reply:
**DOCS**
You can display a group of notifications in a list format, and the user can sort them from the header. To display a notification group, add the `fd-notification fd-notification__group` class.

Reply:
This currently has no description because they are a little misplaced. So, right now this description is under **Error** example. Just be sure to wrap the class names in code, once you've moved them over here.